### DISCUSSION TITLE
add npm create for Feliz template

### DISCUSSION THREAD

**Discussion by jkone27** [Zaid-Ajaj/Feliz#636] [Ideas]
would be great to be able to create the feliz template from npm/npx instread of just dotnet new or just making a quick "alias" behind the scenes?

https://dev.to/mikhaelesa/create-your-own-npm-create-cli-like-create-vite-3ig7

## considerations on `npm create vite` 

Ideally could be great to make a PR to [vite-new repo itself](https://github.com/vitejs/vite/blob/main/packages/create-vite/src/index.ts) to add one option for `fable-feliz-fs` but that probably takes much more effort and for now is best to keep this independent? 
[create-vite-npm](https://www.npmjs.com/package/create-vite)

# potential steps 

1 . check if dotnet-sdk is installed, if not, ask user if ok to download and install dotnet-sdk, or ask him to install it himself providing a clickable link

2 .  run `dotnet new with this template behind the scene`

3 . provide configuration options, for example, for vite 

`* with Vite HMR (experimental)? Y/n` ---> using the awesome [vite-plugin-fable](https://nojaf.com/vite-plugin-fable/)

**Comment by Freymaurer** [comment]
Hey @jkone27 ! I would love to have this.  Maybe you can create a PR for this? 🙂
